### DISCUSSION TITLE
Shot Timer

### DISCUSSION THREAD

**Discussion by Sc00bs110** [rccoleman/lamarzocco#49] [Ideas]
Thanks for the great Home Assistant Integration.

Is there any chance that we would be able to get a shot timer to display in Home Assistant the way it does in the app?

**Comment by rccoleman** [comment]
> Thanks for the great Home Assistant Integration.
> 
> 
> 
> Is there any chance that we would be able to get a shot timer to display in Home Assistant the way it does in the app? 

This integration and the package that supports it no longer work either the recent mandatory firmware updates for the LM machines and you should migrate to https://github.com/zweckj/lamarzocco. It supports a shot clock.

**Reply by Sc00bs110** [reply]
Thank You :-)
